test/Test/HTTPure/TestHelpers.purs

```diff
@@ -76,13 +70,12 @@ toBuffer response =
       stream = HTTPClient.responseAsStream response
     chunks <- Ref.new List.Nil
     Stream.onData stream $ \new -> Ref.modify_ (List.Cons new) chunks
-    Stream.onEnd stream
-      $ Ref.read chunks
+    Stream.onEnd stream $ Ref.read chunks
       >>= List.reverse
-      >>> Array.fromFoldable
-      >>> Buffer.concat
+        >>> Array.fromFoldable
```

Comment:
`purs-tidy` is aware of operator precedence, at least for operators defined in core and contrib libraries; if you would like the operator precedence to be generated for all code the library uses then you can generate an operator table using the `purs-tidy` CLI. I haven't done that here as it seems to be working well!

Reply:
Very nice, good to know, looks great as is though